data/json/npcs/common_chat/TALK_COMMON_ALLY.json

```diff
@@ -1081,5 +1086,85 @@
       { "text": "Yes, I'm sure.", "topic": "TALK_DONE", "effect": "abandon_camp" },
       { "text": "Actually, never mind.", "topic": "TALK_NONE" }
     ]
+  },
+  {
+    "id": [ "TALK_ALLY_DEBUG" ],
+    "type": "talk_topic",
+    "dynamic_line": { "is_by_radio": " *tshk* Not sure it works when i am that far.", "no": "Yes?" },
+    "responses": [
+      {
+        "text": "Trust me (+10 trust).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "trust": 10 } }
+      },
+      {
+        "text": "Do not trust me (-10 trust).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "trust": -10 } }
+      },
+      {
+        "text": "Fear me (+10 fear).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "fear": 10 } }
+      },
+      {
+        "text": "Do not fear me (-10 fear).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "fear": -10 } }
+      },
+      {
+        "text": "Fear me (+10 fear).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "fear": 10 } }
+      },
+      {
+        "text": "Do not fear me (-10 fear).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "fear": -10 } }
+      },
+      {
+        "text": "Value me (+10 value).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "value": 10 } }
+      },
+      {
+        "text": "Do not value me (-10 value).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "value": -10 } }
+      },
+      {
+        "text": "Be angry (+10 anger).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "anger": 10 } }
+      },
+      {
+        "text": "Do not be angry (-10 anger).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "anger": -10 } }
+      },
+      {
+        "text": "Owe me (+10 owed).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "owed": 10 } }
+      },
+      {
+        "text": "Do not owe me (-10 owed).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "owed": -10 } }
+      },
+      {
+        "text": "Favor me (+10 favors).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "favors": 10 } }
+      },
+      {
+        "text": "Do not favor me (-10 favors).",
+        "trial": { "type": "NONE" },
+        "success": { "topic": "TALK_ALLY_DEBUG", "opinion": { "favors": -10 } }
```

Comment:
Is this correct? According to https://github.com/CleverRaven/Cataclysm-DDA/blob/8554ba5309df6681b7bbac710ec64f26ca7f0705/src/savegame_json.cpp#L2106-L2115 there is no such NPC opinion with the name "favors". (Also, not sure what the call to `allow_omitted_members` is supposed to do in that function.)

Reply:
I picked them from NPC.md, yes, and i also don't know where this is used or used at all

Reply:
The documentation appears to be outdated, but favors are still present in npc_favor.h. Might just be dead code. They used to be a vector(?!) stored in the npc.h header, but never within the npc_opinion struct from what I can tell.